spec/features/publish/providers_index_spec.rb

```diff
@@ -117,7 +117,7 @@ def i_can_search_with_provider_details
   end
 
   def i_should_see_the_pagination_link
-    expect(publish_providers_index_page.pagination_pages.text).to eq('2 of 2')
+    expect(publish_providers_index_page.pagination_pages.text).to eq('2 of 4')
```

Comment:
Can we add a step to assert the number of courses within the search results too? 

Or something that might assert `Pagy::DEFAULT[:limit]` or just `10`. 

Keen to hear other thoughts :)